Dockerfile

```diff
@@ -1,4 +1,16 @@
 ARG ARCH=$TARGETARCH
+####################################################################################################
+FROM docker.io/node:18 as numaflow-ui
```

Comment:
We can do the `yarn build` in the `make image` which would expect the user to have yarn installed, we could circumvent it by adding a flag requesting ui vs not requesting it.

Reply:
I think it is not warranted to do this if the docker build is not taking very long

Reply:
👍